kubernetes/configs/dev-refactor/commons.sh

```diff
@@ -57,7 +57,9 @@ PRIVACY_NOTICES_CONTAINER="interop-privacy-notices-content-dev"
 
 PAGOPA_TENANT_ID="69e2865e-65ab-4e48-a638-2037a9ee2ee7"
 
-SMTP_ADDRESS="smtp.gmail.com"
+REPORT_SENDER="noreply@reports.interop.pagopa.it"
```

Comment:
```suggestion
REPORT_SENDER="noreply@reports.refactor.dev.interop.pagopa.it"
```

Reply:
since this is a global variable, may I suggest a rename to
```suggestion
REPORT_MAIL_SENDER="noreply@reports.interop.pagopa.it"
```
(no need to change the env var in the service)